frontend/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM node:18-alpine3.16
+FROM node:18-slim
+#RUN apt-get update && apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb -y
```

Comment:
Can we please remove commented out code?

Reply:
fixed